server/container_status.go

```diff
@@ -135,6 +137,36 @@ func (s *Server) ContainerStatus(ctx context.Context, req *types.ContainerStatus
 	return resp, nil
 }
 
+// hostPathToPossibleImageSpec tries to find an image by the provided host path if the image is mounted locally.
+// It returns nil if nothing was found and logs corresponding errors to not fail unexpectedly.
+func (s *Server) hostPathToPossibleImageSpec(ctx context.Context, hostPath string) *types.ImageSpec {
```

Comment:
do you think we could save this in the volume structure? recalculating on each status may get expensive

Reply:
Yes, done :+1: